packages/iris-grid/src/IrisGridTableModelTemplate.ts

```diff
@@ -455,6 +455,10 @@ class IrisGridTableModelTemplate<
     return !this.isSaveInProgress && this.inputTable != null;
   }
 
+  get isDeletable(): boolean {
+    return this.isDeletable;
+  }
+
```

Comment:
This will cause an infinite loop, since `this.isDeletable` will call the `get isDeletable()` method. Shouldn't need this method anyway since this doesn't implement `DeletableGridModel`.